server/screens.js

```diff
@@ -33,6 +34,58 @@ function generateAdminUpdate(sock) {
 	});
 }
 
+function decideWhichScreenGetsToKeepAnID(screenA, screenB){
+
+	// Emit an event to the screens to reassign using the ID and the timestamp 
+	// of when that id was assigned as the identifier for the screen that needs
+	// to reassign. The original screen (and the rest) can ignore this message.
+	const screenToChange = screenA.idUpdated > screenB.idUpdated ? screenA : screenB ;
+	app.io.of('/screens').emit('reassign', { id : screenToChange.id, idUpdated : screenToChange.idUpdated, newID : generateID() });
+
+}
+
+
+function checkForConflictingId(id){
+
+	return assignedIDs.some(existingScreen => {
+		return existingScreen.id == id;
+	});
+
+}
+
+function checkForConflictingScreens(data){
+
+	return assignedIDs.some(existingScreen => {
```

Comment:
This I had originally (or something very close to it), but I struggle to debug it if it went wrong, so I went for the more verbose option.